### DISCUSSION TITLE
Adding a CLOSE ALL OTHER PLAYERS to the menu

### DISCUSSION THREAD

**Discussion by spambas** [f1multiviewer/issue-tracker#270] [Feature requests] [answered]
I would like an option to close all video players but the one I am currently in

![afbeelding](https://user-images.githubusercontent.com/10828880/198714160-f9ea48d1-986d-42c6-9ac7-2d64b1661dd1.png)

For me only in the main player is fine. 

Like, when the race is over and all cars are parked, the podium ceremony is still on the main channel and I would like to close all OBC's because they don't fill any needs at that time.

**Comment by f1multiviewer** [comment, marked as answer]
Will ship in the next update!
